pkg/sql/grant_revoke.go

```diff
@@ -58,6 +60,27 @@ func (p *planner) changePrivileges(
 	grantees tree.NameList,
 	changePrivilege func(*sqlbase.PrivilegeDescriptor, string),
 ) (planNode, error) {
+	//check whether grantees exists
+	users, err := GetAllUsers(ctx, p)
+
+	if err != nil {
+		return nil, err
+	}
+	users = append(users, security.RootUser)
+	users = append(users, security.NodeUser)
```

Comment:
`node` is hardcoded as allowed to do anything, but it does not get explicit privileges, this can be skipped.